tests/TinyHelpers.Tests/Extensions/CollectionExtensionsTests.cs

```diff
@@ -4,6 +4,9 @@ namespace TinyHelpers.Tests.Extensions;
 
 public class CollectionExtensionsTests
 {
+
+    private IList<int> modifiedList = new List<int>();
```

Comment:
Avoid using an instance variable. Declare it in the arrange part of the test where you're using it.

Reply:
Got it. Thank you for review.

Reply:
Remember to mark the conversation as resolved when you have finished.